meta-iot2000-example/recipes-core/images/iot2000-example-image.inc

```diff
@@ -43,3 +43,6 @@ IMAGE_INSTALL_append = " ofono ofono-tests"
 IMAGE_INSTALL_append = " connman connman-tools connman-tests connman-client"
 
 DEPENDS += "gptfdisk-native efibootguard-native"
+
+# The following is needed for correct EFI deployment
```

Comment:
Yes, but the reason in code would be nice as well : )

Reply:
Sorry, what do you mean by that?

Reply:
You state it's needed. OK, but *why*?

Reply:
Ah sorry... I stumbled across the 'in code'. Since this is just the recipe :)